### ISSUE TITLE
Mac OS X fails to launch docker

### ISSUE THREAD

**Issue by fritzknapp** [docker/kitematic#1485]
[Command failed.docx](https://github.com/docker/kitematic/files/136754/Command.failed.docx)

I am having issue getting docker toolbox 1.10.1 to run on my Mac

Yosemite 10.10.5

When I run <docker version>
Client:
 Version:      1.10.1
 API version:  1.22
 Go version:   go1.5.3
 Git commit:   9e83765
 Built:        Thu Feb 11 19:14:21 2016
 OS/Arch:      darwin/amd64
Cannot connect to the Docker daemon. Is the docker daemon running on this host?

I have tried removing the VM, reinstall... The attached file is output from trying to build the VM through Kitematic

Please let me know what other information you need

**Comment by FrenchBen** [comment]
From your logs it looks like some of the certs may have failed. I would do the following:
Restart your VM:
`docker-machine restart default`

Regenerate your certs:
`docker-machine regenerate-certs default`

**Comment by fritzknapp** [comment]
Ben,

Thanx for the reply, I tried those commands but it didn’t help…

I think that I have narrowed this down to Cisco Anyconnect, I use that to VPN into my work. If I disconnect, remove the VM, and all files (.docker and the files in /usr/local/bin…) reboot and then reinstall, I can get it to work properly…

However as soon as I launch the VPN, it fails and fails every time there after…

Thoughts?


Fritz Knapp
Sr. SAN Administrator
MB Mortgage
a Division of MB Financial
(734) 926-2423 Desk
(734) 255-1378 Cell
fknapp@mbmortgage.com<mailto:fknapp@mbmortgage.com>
